services/Makefile

```diff
@@ -37,7 +37,6 @@ clean-build: clean
 	docker build --network=host --no-cache -t $(IMAGE) -f $(SERVICE)/Dockerfile .
 
 dev:
-	-ln -s $(CURDIR)/shotgrid_common/* $(CURDIR)/$(SERVICE)
```

Comment:
This will break it, even if I change the import to `from shotgrid_common.ayon_shotgrid_hub import AyonShotgridHub`

Reply:
but why should it? isn't `dev` just running image that was build before? And those Dockerfiles are already copying stuff `shotgrid_common` to the `/service`?

Reply:
The way `dev` works is that it mounts the live code (Overriding the copied stuff) so that change can be made and tested without having to rebuild the image

Reply:
Yeah, I get it, but truth to be told, when caching is enabled, the docker image is rebuild in about one second on my slow machine. I am not sure if it is worth the trouble. On Windows it won't work without administrator permissions and if you by accident run docker build while you have at the same time running dev target (with the linked files) it will break the image. At least because of the superfluous __init__.py